src/ast/ddl.rs

```diff
@@ -1296,20 +1296,43 @@ impl fmt::Display for UserDefinedTypeCompositeAttributeDef {
     }
 }
 
-/// PARTITION statement used in ALTER TABLE et al. such as in Hive SQL
+/// PARTITION statement used in ALTER TABLE et al. such as in Hive and ClickHouse SQL
 #[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
-pub struct Partition {
-    pub partitions: Vec<Expr>,
+pub enum Partition {
+    ID(Ident),
+    Expression(Expr),
```

Comment:
```suggestion
    Identifier(Ident),
    Expr(Expr),
```
I think these are more popular variant names in the repo for similar types, figured might be good to try to match them

Reply:
Good call, changed according to the comment.